ipn/conffile/conffile_hujson.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !ios && !android
+
+package conffile
+
+import "github.com/tailscale/hujson"
+
+func init() {
+	hujsonStandardize = hujson.Standardize
+}
```

Comment:
Might be worth a comment on why this is in a different file, so it's local to the file?

Reply:
Done.